memberships/forms.py

```diff
@@ -22,7 +22,11 @@ class RegistrationForm(forms.Form):
     )
     birth_date = forms.DateField(required=True, widget=DateInput)
     constitution_agreed = forms.BooleanField(required=True)
-    donation = forms.DecimalField(min_value=0, decimal_places=2, required=False)
+    constitutional_email = forms.BooleanField(required=True)
+    constitutional_post = forms.BooleanField(required=True)
```

Comment:
This being required forces the user to select these settings but the user can still navigate to their settings and update theis setting to disable them.

Reply:
Yeah I see what you mean, does them turning these off in their settings also mean they should not receive any emails, like a verification email or something

Reply:
I think that these settings should not be edittable in settings as it is an expectation when joining GeekZone (at least that is my understanding of it)